src/parser.rs

```diff
@@ -706,10 +706,15 @@ impl<'a> Parser<'a> {
           self.next(); // consume '}'
           break false;
         }
-        Some((_, c))
-          if had_space && is_name_start(c) && attributes.is_empty() =>
-        {
-          options.push(self.parse_option());
+        Some((_, c)) if had_space && is_name_start(c) => {
+          let option = self.parse_option();
+          if let Some(previous_attribute) = attributes.last() {
+            self.report(Diagnostic::MarkupOptionAfterAttribute {
+              previous_attribute: previous_attribute.clone(),
```

Comment:
Given that the last attribute is whatever comes before the option and not in some other random place, is it actually helpful to keep this for showing diagnostics?

Reply:
It's not necessarially directly before - for example take the case `<attribute> <option> <option>`. I think it will be useful, as the diagnostic can mark both the span that is errored, and then in gray mark the span that is the cause (like rust-analyzer does for mut errors).